.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+Please briefly explain the changes you made here.
```

Comment:
I wouldn't say "briefly". PR descriptions should be thorough enough to explain the functionality and reasons for design choices made. I'd rather a PR author erred on the side of giving too much detail than too little. What do you think?

Reply:
Good shout. Yeah happy with that